modules/services/safeeyes.nix

```
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.safeeyes;
+
+in {
+  meta.maintainers = [ maintainers.rosuavio ];
+
+  options = {
+    services.safeeyes = {
+      enable = mkEnableOption "the safeeyes OSGI service";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.safeeyes;
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    home.packages = [ cfg.package ];
+
+    systemd.user.services.safeeyes= {
+      Install.WantedBy = [ "graphical-session.target" ];
+      Unit = {
+        Description = "Safeeyes";
+        PartOf = [ "graphical-session.target" ];
+        StartLimitIntervalSec = 350;
+        StartLimitBurst = 30;
+        # Path = [ pkgs.alsa-utils ];
```

I think you want something like

```suggestion
        Environment = [ "PATH=${makeBinPath [ pkgs.alsa-utils ]}" ];
```